lib/controller.js

```diff
@@ -91,7 +91,6 @@ Tessel.list = function(opts) {
 
     // When a Tessel is found
     seeker.on('tessel', function displayResults(tessel) {
-
```

Comment:
We generally try to avoid whitespace changes like this to keep the history cleaner.